packages/xo-web/src/common/xo/index.js

```diff
@@ -564,9 +562,14 @@ export const editServer = (server, props) =>
   )
 
 export const enableServer = server =>
-  _call('server.enable', { id: resolveId(server) })::pFinally(
-    subscribeServers.forceRefresh
-  )
+  _call('server.enable', { id: resolveId(server) })
+    ::tapCatch(error => {
+      if (error.message === 'Invalid XML-RPC message') {
+        return error(_('enableServerErrorTitle'), _('enableServerErrorMessage'))
+      }
+      throw error
```

Comment:
You don't need to rethrow the error as the promise will automatically reject with the original error. Also, did you want to *not* reject in case of `Invalid XML-RPC message` error?

Reply:
No, I don't need to not reject it in this case.